### ISSUE TITLE
Chat  - Unable to paste content without formatting using cmd+shift+v

### ISSUE THREAD

**Issue by lanitochka17** [Expensify/App#24259]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Open the staging link
2. Navigate to any chat that has text with formatting
3. Select the formatted text and copy it
4. Go to composer and press cmd+v
5. Press cmd+shift+v

## Expected Result:
Text should be pasted without formatting with pressing cmd+shift+v in composer

## Actual Result:
Nothing happens/ no text is pasted when pressing cmd+shift+v after copying a formatted text

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [ ] MacOS /  Safari
- [ ] MacOS / Desktop

**Version Number:** 1.3.51.0

**Reproducible in staging?:** Yes

**Reproducible in production?:** Yes

**If this was caught during regression testing, add the test name, ID and link from TestRail:**

**Email or phone of affected tester (no customers):**

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation

https://github.com/Expensify/App/assets/78819774/5740f15d-88f8-47a5-9f6f-75fecbfdc044

**Expensify/Expensify Issue URL:**

**Issue reported by:** Applause - Internal Team

**Slack conversation:**

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by akinwale** [comment]
I watched the video, the key combination pressed is not Cmd+Shift+V. The Shift key is the outlined arrow facing up (the first key found on the left hand side or right hand side of the on-screen keyboard, on the second to the last row), while the key pressed in the video in combination with `Cmd` and `V` is a caret / circumflex.

Cmd+Shift+V works as expected.

**Comment by zanyrenney** [comment]
@akinwale are you suggesting this should be closed then? I also can't reproduce with the instructions given. Thanks for your insight!

**Comment by akinwale** [comment]
> @akinwale are you suggesting this should be closed then? I also can't reproduce with the instructions given. Thanks for your insight!

Yes, that's correct. The key combination shown in the video is different from Cmd+Shift+V.

**Comment by zanyrenney** [comment]
Awesome, I agree. Thank you for such a switft response 🙇🏼
